### ISSUE TITLE
File name parse issue. 

### ISSUE THREAD

**Issue by IvanMrmachovski** [igvteam/igv#897]
When accessing a remote file and the url contains the access (GET) parameters and if the parameters contain "/" then the left panel shows a wrong  file name. Instead it shows the random string contained in the parameter after the forward slash "/". Im assuming the code is taking as it was a local file and it parse the last slug as a filename. In this case its a GET parameter that comes from the host for security purpose and as a value sometimes contains random slashes "/". I have screenshots i cant believe this form wont accept pics. Ping me if you need more clarification. Thank you

**Comment by IvanMrmachovski** [comment]
I was able to host the screenshots 
https://ibb.co/zGRsw0X
https://ibb.co/6F5sGcx

**Comment by jrobinso** [comment]
@IvanMrmachovski The screenshot links are spam clickbait, I am deleting that response.   You can upload screesnhots to a git issue.

**Comment by jrobinso** [comment]
@IvanMrmachovski Screenshots aren't neccessary anyway, can you post an example URL.

**Comment by IvanMrmachovski** [comment]
That was light-speed fast. Awesome cheers!
I was just wandering how long its gonna take to get this code into the latest version?

**Comment by jrobinso** [comment]
Early next week.   I think what I pushed will fix it, but if you want me to test one of your URLS post it here.   Here are the forms I've tried.  In all cases the filename is "track"

```
http://foo.bar/track
http://foo.bar/track?param=abc
http://foo.bar/track?param=abc/123"
```

**Comment by IvanMrmachovski** [comment]
For the dear god I couldn't copy the link but I found a good example and made a screenshot for you too see. Now its hosted on google. 
https://photos.app.goo.gl/AKyWJ9B12ja8fWuq9

**Comment by jrobinso** [comment]
BTW this has been released, version 2.9.0
